tensorflow/python/ops/image_ops_impl.py

```diff
@@ -849,7 +849,9 @@ def central_crop(image, central_fraction):
 
   Remove the outer parts of an image but retain the central region of the image
   along each dimension. If we specify central_fraction = 0.5, this function
-  returns the region marked with "X" in the below diagram.
+  returns the region marked with "X" in the below diagram. Larger the value of 
+  central_fraction, larger the dimension of bounding box of the region to be
```

Comment:
"larger the dimension of bounding box of the region to be"

Simplify to "larger the dimension of the region to be"